project-resources/ESP/ECOSYSTEM_SUSTAINABILITY_PROGRAM.md

```diff
@@ -0,0 +1,78 @@
+# OpenJS Ecosystem Sustainability Program 
+
+## Summary 
+
+The Ecosystem Sustainability Program is an opt-in OpenJS partner program to help secure outdated software and create new revenue streams for our projects and commercial participants.  
+
+## Benefits
+
+Eligible projects that opt into the program receive revenue that can be used to:
+
+- Grow their community
+- Pay contractors
+- Support travel
+- Cover other project-related expenses
+
+## Current Partners
+
+- [HeroDevs](https://www.herodevs.com/)
+
+## Eligibility
+
+For a project to participate in the program:
+
+- A partner must be providing support to EOL versions of the project
+- The project must be willing to place links to the partner on various pages/repositories where EOL versions are mentioned (requirements will vary per partner)
```

Comment:
How does the “requirements will vary per partner” work in practice? It feels like this should be negotiated by the foundation for all of the projects and partners and not the fruit of 1:1 negotiations between projects and partners.

Reply:
Also worth linking to the section below where more info is provided.